lib/utils/fsUtils.js

```diff
@@ -69,7 +81,7 @@ async function createLiquidTestYaml(templateType, handle, testContent) {
     if (error) {
       errorCallbackLiquidTest(error);
     } else {
-      console.log(`Liquid testing YAML file created for ${handle}`);
+      consola.debug(`Liquid testing YAML file created for ${handle}`);
```

Comment:
Could also still be useful as info imo

Reply:
In that case it will always be shown. By default we show info. I think once you know it will create those files, you would improve your experience reducing the terminal output (it won't add any value to you)